test/e2e/storage/utils/create.go

```diff
@@ -67,6 +68,13 @@ func LoadFromManifests(files ...string) ([]interface{}, error) {
 			return nil
 		}
 
+		// Sections of entirely comments get treated as an "object", such as the beginning of
+		// test/e2e/testing-manifests/storage-csi/external-snapshotter/csi-snapshotter/rbac-csi-snapshotter.yaml
+		// If the object's Kind is empty, assume it is one of these dummy objects and skip it
+		if what.Kind == "" {
```

Comment:
Please remove it. It's already handled in line 67.

Reply:
Removed, someone else apparently beat me to it while this PR was in-flgiht